### DISCUSSION TITLE
Ask opening hours for recyling containers?

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#2851] [Ideas]
For example many battery recycling points (in Poland) are inside shops etc so have opening hours.

The same for recycling containers of small appliances.

In some places there are opening hours for ones outside, due to noise generated by dropping for example glass into them ( https://github.com/simonpoole/beautified-JOSM-preset/issues/111 )

Though @westnordost commented 

> Though, the "opening hours" are usually all the same, so the information value of explicitly tagging this (on every container individually) is quite low.

in https://github.com/simonpoole/beautified-JOSM-preset/issues/111

So maybe ask only about indoor ones? (how many are even tagged with `location=indoor` - maybe ask about that?) EDIT: just 57 worldwide, see http://overpass-turbo.eu/s/1eji

Or is it expected that for say https://www.openstreetmap.org/node/8638559184 to be expected to be somehow linked to shop operating them (in this case Lidl) rather than repeating shop opening hours?

**Comment by HolgerJeromin** [comment]
Situation in Germany:
"All" `glass_bottles` and `clothes` container are outdoor and the `glass_bottles` have opening hours signed (clothes do not).
I do not know a single `batteries` container which is outdoor or 24/7, because it 
* is placed on a table inside the shop beneath the cash register or
* is part of a recycling_type=centre with own opening hours

**Reply by westnordost** [reply]
But they all have the same opening hours, no?

**Reply by riQQ** [reply]
CityGreenPoint Deutschland GmbH has battery recycling that is outdoors and available 24/7:
http://www.citygreenpoint.nl/en/products-services.html

**Reply by HolgerJeromin** [reply]
> But they all have the same opening hours, no?

In the sense of: not at night. Yes.
But I am not sure if they are all 7-20 like the ones in my area.

**Reply by matkoniecz** [reply]
From quick check at http://overpass-turbo.eu/s/172i it seems that oh vary even if in a given region like Berlin some are dominant

https://www.openstreetmap.org/node/8131353309
https://www.openstreetmap.org/node/7413155782
https://www.openstreetmap.org/node/7863698607
https://www.openstreetmap.org/node/7872526826

**Comment by matkoniecz** [comment]
https://github.com/streetcomplete/StreetComplete/commit/c133bfca1508be1040125330e1895bf02faffd85

![screen13](https://user-images.githubusercontent.com/899988/146736235-4cb763f0-2788-43cd-9bfa-74e0956fb549.png)
https://www.openstreetmap.org/node/8638559184
